tools/owncloud-unlockadmin.sh

```diff
@@ -4,20 +4,20 @@
 # instance running here.
 #
 # Run this at your own risk. This is for testing & experimentation
-# purpopses only. After this point you are on your own.
+# purposes only. After this point you are on your own.
 
 source /etc/mailinabox.conf # load global vars
 
 ADMIN=$(./mail.py user admins | head -n 1)
 test -z "$1" || ADMIN=$1 
 
-echo I am going to unlock admin features for $ADMIN.
-echo You can provide another user to unlock as the first argument of this script.
+echo "I am going to unlock admin features for $ADMIN."
+echo "You can provide another user to unlock as the first argument of this script."
 echo
-echo WARNING: you could break mail-in-a-box when fiddling around with Nextcloud\'s admin interface
-echo If in doubt, press CTRL-C to cancel.
+echo "WARNING: you could break mail-in-a-box when fiddling around with Nextcloud\'s admin interface"
+echo "If in doubt, press CTRL-C to cancel."
 echo 
-echo Press enter to continue.
+echo "Press enter to continue."
```

Comment:
Perhaps take the opportunity to capitalise to `Enter`:
```suggestion
echo "Press Enter to continue."
```

Reply:
I was trying to keep this pull request to bugs found by ShellCheck and a few minor spelling errors, per your comment: https://github.com/mail-in-a-box/mailinabox/pull/1435#issuecomment-429839077.